test-fixture/src/sim/rng.rs

```diff
@@ -29,6 +30,12 @@ impl Random {
         }
     }
 
+    #[cfg(feature = "disable-random")]
+    pub fn random(&mut self) -> u64 {
+        0
+    }
```

Comment:
I think that it would be better if you used a fixed seed (or one controlled by the fuzzer) instead.

Reply:
I'm not sure if the fuzzer would be as successful in mutating the practically random (but fixed) output that would generate vs. zimple zeros?

Reply:
I changed that in 376686814eda359fb17f8067a0fd744d98bd9708, please take a look?

(It does have the nice side effect that way more tests now pass, generating a much larger corpus for the fuzzer to then mutate.)